src/wrappedContracts/CertificateLogic.ts

```diff
@@ -631,12 +631,12 @@ export class CertificateLogic extends GeneralFunctions {
         }
     }
 
-    async createTradableEntity(_assetId: number, _powerInW: number, txParams?: SpecialTx) {
+    async requestCertificates(_assetId: number, limitingSmartMeterReadIndex: number, txParams?: SpecialTx) {
         let transactionParams;
 
-        const txData = await this.web3Contract.methods
-            .createTradableEntity(_assetId, _powerInW)
-            .encodeABI();
+        const preparedMethod = this.web3Contract.methods.requestCertificates(_assetId, limitingSmartMeterReadIndex)
```

Comment:
Maybe rename `preparedMethod()` to something more descriptive? `preparedMethod` name doesn't give us a hint which method is being prepared by it.